shell_scripts/worker.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "qworker is starting..."
+
+# Wait for backend to be ready since supervisor will spawn all the 3 processes backend, qworker, and frontend same time
+while [ ! -f /tmp/backend_ready ]; do
```

Comment:
I'd think if this needs to wait, it could use the same pattern to check for the database and just look for localhost and the port, then it wouldn't need a temp file?

We could leave this and think about it more as we go toward production, whether this worker would run in the same pod, a separate pod or as a scheduled task. 

```
if [ -z "${GUNICORN_PORT}" ]; then
    GUNICORN_PORT=4001
fi 
```

Reply:
The Reason why this is there is because during initial setting up...
1. qworker need access to its DB tables so migrations needs to happen
2. If Backend is starting up and so is qworkers and migration are not done then you will see `django_q_ormq` table is not available. 
3. so it's not just DB availability but running through all migration. Initially that's what I was doing checking DB host/ports are available or not, but that check was not enough.

Reply:
>We could leave this and think about it more as we go toward production, whether this worker would run in the same pod, a separate pod or as a scheduled task.

I am open to this idea during prod